feat(documentation) First draft of the October newsletter. (#5666)
jenkins-infra/jenkins.io: content/blog/2022/11/2022-11-04-jenkins-newsletter.adoc

** The Security team did an investigation and nothing in the Jenkins ecosystem is affected
** Due to the https://commons.apache.org/proper/commons-text/security.html[communication from Apache] about why it’s not a vulnerability like Log4Shell&#44; we do not plan to publish a blog post about our analysis.

== Infrastructure Update 

* Hacktoberfest support: ~15% more builds for a cost decrease of $2.000 emoji:clap[]
* Fall cleanup of accounts.jenkins.io and wiki.jenkins.io (Unified designs, dependencies updates, bug fixes)
* Upgrade of our controllers to the new LTS 2.361.2
* Upgrade of Kubernetes to 1.23.x for the 6 clusters (Amazon, Digital Ocean and Azure)
* Request for volunteers to host download mirrors in India
* Security Advisories support (Jenkins Infrastructure is the first user!)

== Platform Update

* Very last JDK8 https://github.com/jenkinsci/docker-inbound-agent/pull/292[docker images] have been released
* https://github.com/jenkinsci/docker-ssh-agent/pull/165[Preparation] of the three agents repos merging while solving a https://github.com/jenkinsci/docker-ssh-agent/issues/131[bug].

> * Lost of the ppc64le loaner machine from IBM, some reference https://github.com/jenkins-infra/helpdesk/issues/3198[cleaning] to do.

Reviewer: ```suggestion
* Loss of the ppc64le loaner machine from IBM, some reference https://github.com/jenkins-infra/helpdesk/issues/3198[cleaning] to do.
```
small syntax suggestion

> * Lost of the ppc64le loaner machine from IBM, some reference https://github.com/jenkins-infra/helpdesk/issues/3198[cleaning] to do.

Reviewer: ```suggestion
* Loss of the ppc64le loaner machine from IBM, some reference https://github.com/jenkins-infra/helpdesk/issues/3198[cleaning] to do.
```
small syntax suggestion

* JDK17 aarch64 docker image bug https://github.com/jenkinsci/docker-agent/issues/308[discovered] and https://github.com/jenkinsci/docker-agent/pull/309[solved].
* First https://riscv.org/[RISC-V] https://mangopi.org/mqpro[machine] received, tests should happen in the coming months
* JVM stats for October: Java 8 fell from 153,205 to 139,827; Java 11 rose from 137,332 to 141,387; and Java 17 rose from 11,173 to 14,420. There are now more users on Java 11 than Java 8.

== User Experience Update

The http://accounts.jenkins.io[accounts.jenkins.io] page has been significantly updated thanks to the work of Tim Jacomb, Alex Brandes, and Gavin Mogan.

> At a high level, Tim and Alex made major dependency updates. Tim was also able to update the UI to be more modern and synchronize with the http://jenkins.io[jenkins.io] interface. Gavin was able to create and add the page header & footer, so that page transitions are seamless. This work to keep Jenkins aligned and up to date is greatly appreciated, and necessary to keep it modern.

Reviewer: ```suggestion
At a high level, Tim and Alex made major dependency updates.
Tim was also able to update the UI to be more modern and synchronize with the http://jenkins.io[jenkins.io] interface.
Gavin was able to create and add the page header & footer, so that page transitions are seamless.
This work to keep Jenkins aligned and up to date is greatly appreciated, and necessary to keep it modern.
```
formatting for sentence per line

image:/images/post-images/2022-11-04-jenkins-newsletter/image10.png[image,width=416,height=272]

> The http://wiki.jenkins.io[old Jenkins wiki] site has also received a face lift, as it has been updated with the stylesheets used for the accounts page update. Despite the wiki being deprecated, finding and reading information on the site is much easier. This is yet another way to make the page transitions more seamless throughout Jenkins.

Reviewer: ```suggestion
The http://wiki.jenkins.io[old Jenkins wiki] site has also received a face lift, as it has been updated with the stylesheets used for the accounts page update.
Despite the wiki being deprecated, finding and reading information on the site is much easier.
This is yet another way to make the page transitions more seamless throughout Jenkins.
```
formatting suggestion for sentence per line

image:/images/post-images/2022-11-04-jenkins-newsletter/image12.png[image,width=407,height=258]

== Documentation Update

image:/images/post-images/2022-11-04-jenkins-newsletter/image5.png[image,width=236,height=132] The annual Jenkins board members and officers election is here!
Thanks https://www.jenkins.io/blog/authors/kmartens27/[Kevin Martens] for thoroughly laying out the https://www.jenkins.io/blog/2022/10/20/jenkins-election-announcement/[voter registration and nomination process].
Nominate yourself or someone you know. But hurry because nominations end on November 10, 2022.

> Can’t think of anyone to nominate? We’d gladly take your vote! https://community.jenkins.io/g/election-voter-2022[Voter registration] is open now until November 17. The voting period will begin on November 17 - December 2, 2022. Election process page in Jenkins will be updated accordingly.

Reviewer: ```suggestion
Can’t think of anyone to nominate? 
We’d gladly take your vote!
https://community.jenkins.io/g/election-voter-2022[Voter registration] is open now until November 17.
The voting period will begin on November 17 - December 2, 2022.
The election process page in Jenkins has been updated accordingly.
```
formatting suggestion for one sentence per line, small wording update as this page has been updated

Jenkins GSoC 2022 ended in October with the successful completion of all 4 projects. Below are the final reporting from each projects:

* https://www.jenkins.io/blog/2022/10/10/pipeline-steps-improvement-gsoc-report/[Pipeline Steps Documentation Generator Improvements] by https://www.jenkins.io/blog/authors/vihaanthora/[Vihaan Thora]
* https://www.jenkins.io/blog/2022/10/10/plugin-health-scoring-system-report/[Plugin Health Scoring System] by https://www.jenkins.io/blog/authors/dheerajodha/[Dheeraj Singh Johna]
* https://www.jenkins.io/blog/2022/09/07/jenkinsfile-runner-as-github-actions/[Jenkinsfile Runner Action for GitHub Actions] by https://www.jenkins.io/blog/authors/yiminggong/[Yiming Gong]
* https://www.jenkins.io/projects/gsoc/2022/projects/automatic-git-cache-maintenance/[Automatic Git Cache Maintenance on the Controller] by https://www.jenkins.io/blog/authors/hrushikeshrao/[Hrushikesh Rao]

Demos were represented during the Jenkins Online Meetup. Recording is https://youtu.be/fM2SMbppRxw[HERE].

This program isn’t possible without the dedication of all the mentors. Hats off to all Jenkins GSoC mentors!

=== Hacktoberfest

> image:/images/post-images/2022-11-04-jenkins-newsletter/image6.png[image,width=81,height=118] Participation in the 2022 edition of this worldwide event was strong. 117 seasoned but also first-time contributors submitted 613 eligible PRs. From these, 531 PRs are "Hacktoberfest complete" (merged or flagged as hacktoberfest-approved). They were submitted by 95 contributors (among them 42 qualify for the swag just with Jenkins contribution).

Reviewer: ```suggestion
image:/images/post-images/2022-11-04-jenkins-newsletter/image6.png[image,width=81,height=118]
Participation in the 2022 edition of this worldwide event was strong.
117 seasoned but also first-time contributors submitted 613 eligible PRs.
From these, 531 PRs are "Hacktoberfest complete" (merged or flagged as hacktoberfest-approved).
They were submitted by 95 contributors (among them 42 qualify for the swag just with Jenkins contribution).
```
small formatting suggestion for formatting
